flask_sqlalchemy/__init__.py

```diff
@@ -874,8 +874,13 @@ def apply_driver_hacks(self, app, sa_url, options):
         if sa_url.drivername.startswith('mysql'):
             sa_url.query.setdefault('charset', 'utf8')
             if sa_url.drivername != 'mysql+gaerdbms':
-                options.setdefault('pool_size', 10)
-                options.setdefault('pool_recycle', 7200)
+                pool_class_name = None
+                if 'poolclass' in options:
```

Comment:
`pool_class` needs an underscore here.

Reply:
If we scroll down a few lines there are examples of no underscore: `options['poolclass'] = StaticPool`.

I appreciate this PR has mixed both underscores and no underscores, so my question is, which should it be, and does the rest of the code need fixing?

Reply:
Never mind, it's the name of the argument in SQLAlchemy: https://docs.sqlalchemy.org/en/13/core/engines.html#sqlalchemy.create_engine.params.poolclass.

Which means the next line needs to be `options['poolclass']`.

Reply:
Done :)